locales-pending/landing-all.ftl

```diff
@@ -20,3 +20,22 @@ landing-all-value-prop-info-at-risk-description = Data leaks are unfortunately p
 # Quote
 
 landing-all-quote = <data_breaches>Data breaches</data_breaches> happen every 11 minutes, exposing your private information — but don’t worry, we can help.
+
+# Social proof
+
+# Variables
+# $num_users is the number of users in the millions.
+landing-all-social-proof-title = Trusted by { $num_users } million people worldwide
```

Comment:
```suggestion
landing-all-social-proof-title = { $num_users ->
   *[other] Trusted by { $num_users } million people worldwide
}
```

I know it feels weird as an English speaker, but most language pluralize "million". This would trigger the plural UI in Pontoon, and it doesn't have a practical effect on English compared to the existing text.